xbmc/GUIPassword.cpp

```diff
@@ -118,6 +109,26 @@ bool CGUIPassword::IsItemUnlocked(CFileItem* pItem, const std::string &strType)
   return true;
 }
 
+bool CGUIPassword::IsItemUnlocked(CFileItem* pItem, const std::string &strType)
+{
+  const std::string strLabel = pItem->GetLabel();
+  std::string strHeading = "";
```

Comment:
No need to explicitly initialize to an empty string; `std::string strHeading;` is preferred.

Reply:
done